rosdep/python.yaml

```diff
@@ -1797,6 +1797,52 @@ python-hypothesis:
     yakkety_python3: [python3-hypothesis]
     zesty: [python-hypothesis]
     zesty_python3: [python3-hypothesis]
+python-imageio:
+  alpine:
+    pip:
+      packages: [imageio]
+  arch:
+    pip:
+      packages: [imageio]
+  cygwin:
+    pip:
+      packages: [imageio]
+  debian:
+    '*':
+      pip:
+        packages: [imageio]
+    buster: [python-imageio]
+  fedora:
+    pip:
+      packages: [imageio]
+  freebsd:
+    pip:
+      packages: [imageio]
+  gentoo: [dev-python/imageio]
+  macports:
+    pip:
+      packages: [imageio]
+  opensuse:
+    '*':
```

Comment:
Same logic reversal here. We don't want to accidentially have new users install the pip version on newer systems. The wildcard should be used for the expected future releases so as not to need to be updated.